content/en/docs/clients/restful-api/propertykey.md

```
@@ -121,7 +121,7 @@ PUT http://localhost:8080/graphs/hugegraph/schema/propertykeys/age?action=append
 ##### Method & Url
 
 ```
-GET http://localhost:8080/graphs/hugegraph/schema/propertykeys
+GET http://localhost:8080/graphspaces/DEFAULT/graphs/hugegraph/schema/propertykeys
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)